app/src/main/res/values/picover_logo_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
i am not sure about the naming, maybe it will be better to stay with the icon specific file names, like:
- ic_launcher
- ic_launcher_background
- ic_launcher_foreground 
but let's check the others opinion :)

Reply:
Yes, it is more generic and I like this idea so I will go with it 😄